scripts/sandbox.ts

```diff
@@ -246,16 +246,31 @@ function addEsbuildLoaderToStories(mainConfig: ConfigFile) {
   );
 }
 
-// Recompile optimized deps on each startup, so you can change @storybook/* packages and not
-// have to clear caches.
-function forceViteRebuilds(mainConfig: ConfigFile) {
+/*
+  Recompile optimized deps on each startup, so you can change @storybook/* packages and not
+  have to clear caches.
+  And allow "template-stories" directory if necessary
+
+*/
+function setSandboxViteFinal(mainConfig: ConfigFile) {
   const viteFinalCode = `
   (config) => ({
     ...config,
     optimizeDeps: {
       ...config.optimizeDeps,
       force: true,
     },
+    plugins: [
+      ...config.plugins,
+      {
+        name: 'storybook:allow-template-stories',
+        config(config) {
+          if (config?.server?.fs?.allow) {
+            config.server.fs.allow.push('template-stories');
```

Comment:
yeah I tried that but you actually can't because that won't pick up on other plugin's configs.
So if I set it directly on the config `config?.server?.fs?.allow` will be falsy, while doing it through this plugin logic `config?.server?.fs?.allow` will actually include `vite-plugin-svelte-kit`'s allow list.

Reply:
I meant without the check, you could just set the allow value directly, and then the rest of everything should still work.  `vite-plugin-svelte-kit` should extend the user's config and the other plugin you created should as well, right?  If we can't set custom allow paths in `viteFinal`, we're in trouble.

Reply:
I think the plugin is probably the safer approach because it can check whether the user has set `server.fs.allow` themselves or not. If we always pass `server.fs.allow: ['.storybook']` then we enable that feature which will may break the user's project if they have not added the directories their project uses to the config.

Reply:
In this sandbox, we _are_ the user.  I'm not talking about setting it in all cases, just in the example sandbox, which is used for testing.

Reply:
oh yeah, that sounds like the better approach then